integration_tests/test_dict_keys_values.py

```diff
@@ -50,5 +50,26 @@ def test_dict_keys_values():
                 key_count += 1
                 assert v2_copy[j] == d2[str(i)]
         assert key_count == 1
+    
+    # dict.values on dict constant
+    assert len({1: "a"}.values()) == 1
+    print({1: "a"}.values())
```

Comment:
Instead of just checking the `len`, can you instead do `list({1: "a"}.values()) == ["a"]`? This way it is a robust test as it checks all the individual elements.

Reply:
Please see https://github.com/lcompilers/lpython/pull/2660#discussion_r1580475206